react/components/EditorContainer/Sidebar/ComponentEditor/Form.tsx

```diff
@@ -21,7 +21,7 @@ export const widgets = {
   RadioWidget: Radio as Widget,
   SelectWidget: Dropdown as Widget,
   TextareaWidget: TextArea as Widget,
-  'image-uploader': ImageUploader as Widget,
+  'image-uploader': (ImageUploader as unknown) as Widget,
```

Comment:
:eyes:

Reply:
Me after many failed attempts at fixing this:
![imposibru](https://user-images.githubusercontent.com/10400340/58519251-1d161580-8189-11e9-8c14-9c8cf144aea0.jpg)